js_modules/dagit/packages/ui/src/components/Icon.tsx

```diff
@@ -330,15 +326,16 @@ export const IconWrapper = styled.div<WrapperProps>`
         // We could just use !important but specificity is a little more flexible
         `
         background: url(${p.$img});
+        background-size: cover;
         &[role='img'][role='img'] {
           background-color: transparent;
         }
       `
       : `
         background: ${p.$color};
+        mask-size: cover;
         mask-image: url(${p.$img});
       `}
-  mask-size: cover;
```

Comment:
is this needed? You're not using <Icon/> anymore so I think not?

Reply:
I think when you specify a "fill color" we're using a solid color + mask image, so the mask-size is just necessary in that case. Jamie and I moved these rules up into the ternary above!